pkg/auth/azure.go

```diff
@@ -44,6 +49,24 @@ type User struct {
 	Expires time.Time
 }
 
+type TokenResponse struct {
+	AccessToken string `json:"access_token"`
+}
+
+type MemberOfResponse struct {
+	Groups []MemberOfGroup `json:"value"`
+}
+
+type MemberOfGroup struct {
+	DisplayName string   `json:"displayName"`
+	Mail        string   `json:"mail"`
+	GroupTypes  []string `json:"groupTypes"`
+}
+
+const (
+	AzureGraphMemberOfEndpoint = "https://graph.microsoft.com/v1.0/me/memberOf/microsoft.graph.group?$select=mail,groupTypes,displayName"
```

Comment:
Why do we need all three of these values? Isn't it enough with mail?

Reply:
ok